package.json

```diff
@@ -12,54 +11,85 @@
     "url": "https://github.com/sanity-io/hierarchical-document-list/issues"
   },
   "homepage": "https://github.com/sanity-io/hierarchical-document-list#readme",
+
+  "description": "Sanity Plugin - Hierarchical Document List",
+  "keywords": [
+    "sanity",
+    "sanity-plugin"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "source": "./src/index.ts",
+      "require": "./dist/index.js",
+      "import": "./dist/index.esm.js",
+      "default": "./dist/index.esm.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.esm.js",
+  "source": "./src/index.ts",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "sanity.json",
+    "src",
+    "v2-incompatible.js",
+    "package.json"
+  ],
   "scripts": {
-    "dev": "tsc -w -d",
-    "build": "tsc -d",
-    "format": "prettier src -w",
-    "lint": "eslint src",
-    "prepare": "npm run build && husky install",
```

Comment:
did you intentionally remove this prepare hook and husky install?  the husky hooks still exist in the `.husky` folder in the repo but they won't ever be called on new installs nor will the corresponding linting actions be run...